conf/evolutions/reversions/022-add-foreign-datastore.sql

```diff
@@ -2,4 +2,6 @@ START TRANSACTION;
 DROP VIEW webknossos.datastores_;
 ALTER TABLE webknossos.datastores DROP COLUMN isForeign;
 CREATE VIEW webknossos.dataStores_ AS SELECT * FROM webknossos.dataStores WHERE NOT isDeleted;
+
+UPDATE webknossos.releaseInformation set schemaVersion = 21;
```

Comment:
probably slipped in there

Reply:
so the reversion has to set it back to 21. (this was missing in #3080 )